adapters/airflow_connector.py

```diff
@@ -24,33 +25,60 @@ def query(self, request):
     Returns:
         The database result (tuple)
     """
-    cursor = self._airflow_db.cursor()
     # The following ensures that the query executes and returns,
     # even if the db connection has been lost
+    logging.info(request)
     try:
+      cursor = self._airflow_db.cursor()
       cursor.execute(request)
       response = cursor.fetchall()
     except MySQLdb.OperationalError, e:
-      if e[0] == 2006:
+      logging.error('Error %i triggered with MySQLdb: %s'  % (e[0], e[1]))
+      if e[0] in [2006, 2013]:
```

Comment:
Please error must be declared globally on the mysql module, please use the global variable instead of those numbers. 
```python
from mysql.connector import errorcode
...
if e[0] in [errorcode.<>, errorcode.<>]
```